drivers/wifi/nrf700x/zephyr/src/zephyr_wifi_mgmt.c

```diff
@@ -796,3 +796,73 @@ int wifi_nrf_set_power_save_timeout(const struct device *dev,
 out:
 	return ret;
 }
+
+#ifdef CONFIG_WIFI_FEAT_RAW_SCAN_RESULTS
+void wifi_nrf_rx_bcn_prb_resp_frm(void *vif_ctx,
+				  void *nwb,
+				  unsigned short frequency,
+				  signed short signal)
+{
+        struct wifi_nrf_vif_ctx_zep *vif_ctx_zep = vif_ctx;
+	struct wifi_nrf_ctx_zep *rpu_ctx_zep = NULL;
+	struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx = NULL;
+	struct wifi_raw_scan_result *bcn_prb_resp_info = NULL;
+	int frame_length = 0;
+	int val = signal;
+
+	vif_ctx_zep = vif_ctx;
+
+	if (!vif_ctx_zep) {
+		LOG_ERR("%s: vif_ctx_zep is NULL\n", __func__);
+		return;
+	}
+
+	if (!vif_ctx_zep->scan_in_progresss) {
```

Comment:
We can have a information log message here.

Reply:
In case it's associated it will keep on printing the message. 
Kept under commented code.